srcpkgs/sway/template

```diff
@@ -1,6 +1,7 @@
 # Template file for 'sway'
 pkgname=sway
-version=1.7
+_rc="1"
+version=1.8
```

Comment:
(I can not comment in the right place, so I will leave the comment here)

* replace pcre-devel with pcre2-devel

Reply:
Done